lib/finder_game/const/finder_constraints.dart

```diff
@@ -5,9 +5,17 @@ class FinderConstraints {
   static const trashAdditionalTopPaddingPercentage = 0.08;
   static const trashSizeFactor = 0.15;
   static const sidePadding = 5;
-
+  static const kVerticalHolePositionOffset = 56;
+  static const kHoleZoomFactor = 0.2;
+  static const timerTicksLimit = 3;
```

Comment:
From my point of view the naming of the constant variables looks strange, what was the reason to add k suffix to  `kVerticalHolePositionOffset` and `kHoleZoomFactor`?

Reply:
renamed